src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -332,6 +332,18 @@ def prepare_or_run_pipeline(
                     else None
                 ),
             )
+            pipeline_tags = step_settings.pipeline_tags
+            args_for_step_executor.setdefault(
```

Comment:
Why are we passing these here as well? I couldn't find an argument `pipeline_tags` for the estimator class that this gets passed to.

Reply:
oh wait i thought it was doing something completely different, but I removed this now